<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chromatic aberration corrector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="inputGroupBox">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Localization file</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="locFileEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="locFileButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="inputTabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="splitOneColorTab">
         <attribute name="title">
          <string>split one-color</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Left ROI</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="3" column="2">
              <widget class="QSpinBox" name="leftRoiTlY">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QSpinBox" name="leftRoiBrX">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="leftRoiTlX">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>x</string>
               </property>
              </widget>
             </item>
             <item row="3" column="5">
              <widget class="QSpinBox" name="leftRoiBrY">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Bottom right</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>y</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Top left</string>
               </property>
              </widget>
             </item>
             <item row="0" column="6">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Right ROI</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="3" column="2">
              <widget class="QSpinBox" name="rightRoiTlY">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QSpinBox" name="rightRoiBrX">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="rightRoiTlX">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>x</string>
               </property>
              </widget>
             </item>
             <item row="3" column="5">
              <widget class="QSpinBox" name="rightRoiBrY">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Bottom right</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>y</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Top left</string>
               </property>
              </widget>
             </item>
             <item row="0" column="6">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="twoOneColorTab">
         <attribute name="title">
          <string>two one-color</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Localization file (channel 2)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="locFile2Edit"/>
            </item>
            <item>
             <widget class="QToolButton" name="locFile2Button">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset theme="document-open">
                <normaloff/>
               </iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="twoColorTab">
         <attribute name="title">
          <string>two-color</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Nothing further to do</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="trafoGroupBox">
     <property name="title">
      <string>Find transformation</string>
     </property>
     <layout class="QVBoxLayout" name="trafoLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Relative tolerance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rTolBox">
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Absolute tolerance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="aTolBox">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="determineButton">
        <property name="text">
         <string>Determine transformation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="text">
        <string>Save results</string>
       </property>
       <property name="icon">
        <iconset theme="document-save-as">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>locFileEdit</tabstop>
  <tabstop>locFileButton</tabstop>
  <tabstop>inputTabWidget</tabstop>
  <tabstop>leftRoiTlX</tabstop>
  <tabstop>leftRoiTlY</tabstop>
  <tabstop>leftRoiBrX</tabstop>
  <tabstop>leftRoiBrY</tabstop>
  <tabstop>rightRoiTlX</tabstop>
  <tabstop>rightRoiTlY</tabstop>
  <tabstop>rightRoiBrX</tabstop>
  <tabstop>rightRoiBrY</tabstop>
  <tabstop>rTolBox</tabstop>
  <tabstop>aTolBox</tabstop>
  <tabstop>determineButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>locFile2Edit</tabstop>
  <tabstop>locFile2Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
